### DISCUSSION TITLE
Show discussions in issue/prs list

### DISCUSSION THREAD

**Discussion by Nixinova** [lowlighter/metrics#323] [Ideas]
Maybe this plugin should list discussions as well

![image](https://user-images.githubusercontent.com/42429413/119249442-b77c8880-bbec-11eb-8b1b-28eee367f1ed.png)

**Comment by lowlighter** [comment]
Yes it would be pretty nice to include discussions stats 🙂

I know the GraphQL API for discussion is available through an additional header but I didn't dig in it too much. It's not on the graphql explorer yet  but there are some docs iirc.

I can see it be pretty suitable for `followup`, but I'm wondering wether it should contains only Q/A discussions (which can be solved) or any kind of discussions

**Reply by Nixinova** [reply]
I think it should be split into answered/unanswered/unanswerable 
(colors `#55ff55`/`#0055ff`/`#555555` or something)

**Comment by lowlighter** [comment]
This was shipped with #430 through the `discussions` plugin 🎉
